### PR TITLE
Add FLOFO (Fast LOFO)

### PR DESCRIPTION
**Pull request by aerdem4**

### CODE REVIEW

File: lofo/lofo_importance.py [hunk 1]
```diff
@@ -55,7 +54,6 @@ def get_importance(self):
             manager = multiprocessing.Manager()
             result_queue = manager.Queue()
 
-            base_cv_score = self._get_cv_score_parallel('all', self.features, result_queue, True)
```
Comment (stephanecollot) [inline lofo/lofo_importance.py]: hum why this is deleted?

File: lofo/flofo_importance.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        return scores
+
+    def _run_parallel(self, feature_name, n, result_queue):
+        test_score = self._run(feature_name, n)
+        result_queue.put((feature_name, test_score))
+        return test_score
+
+    def get_importance(self):
+        base_score = self._get_score(self.df)
+        n = 10
```
Comment (stephanecollot) [inline lofo/flofo_importance.py]: should be parameter

File: lofo/flofo_importance.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+        self._bin_features()
+
+    def _bin_features(self):
+        self.bin_df = pd.DataFrame()
+        for feature in self.features:
+            self.bin_df[feature] = (self.df[feature].rank(pct=True)*(10 - 0.001)).astype(int)
+
+    def _get_score(self, df):
+        return self.scoring._sign*self.scoring._score_func(self.predict_func(df[self.features]), self.df[self.target])
```
Comment (stephanecollot) [inline lofo/flofo_importance.py]: so no scikit-learn scoring here?

Comment (stephanecollot) [reply lofo/flofo_importance.py]: actually this is function expect this one
https://github.com/scikit-learn/scikit-learn/blob/0e9520b235522146b7395b6d848319d3810aba97/sklearn/model_selection/_validation.py#L585

Comment (stephanecollot) [reply lofo/flofo_importance.py]: Or maybe we could use 
https://github.com/scikit-learn/scikit-learn/blob/8a4cb4871d07c0ddd966fce0959d8118a8f62c59/sklearn/metrics/scorer.py#L243


```
scorer = check_scoring(estimator=estimator, scoring=scoring)
score = scorer(estimator, X_test, y_test)
```

File: lofo/lofo_importance.py (hunk 1 above)
Comment (aerdem4) [reply lofo/lofo_importance.py]: It was redundant work (see line 50)

File: lofo/flofo_importance.py (hunk 2 above)
Comment (aerdem4) [reply lofo/flofo_importance.py]: indeed

**Pull request merged**